fxa/tests/test_oauth.py

```diff
@@ -624,6 +624,10 @@ def callback(self, request):
             return (200, {}, self.body)
         return (500, {}, '{}')
 
+    def _make_jwt(self, payload, key, alg="RS256", header={"typ": "at+jwt"}):
+        header = header.copy()  # So we don't accidentally mutate argument in-place
+        return six.ensure_text(jwt.encode(payload, key, alg, header))
```

Comment:
Previously this was doing `str(jwt.encode(...))` which on python3, results in a string like `"b'jwt-blah-bah-here'"` and was causing the test failures.

Reply:
You can tell I never wrote a line of python3 in my life 😂